Microsoft.Toolkit.Uwp.UI.Controls/QuickReturnHeader/QuickReturnHeader.cs

```diff
@@ -0,0 +1,275 @@
+            {
+                me.StopAnimation();
+
+                if (me.IsQuickReturnEnabled)
+                {
+                    me.StartAnimation();
+                }
+            }
+        }
+
+        private void QuickReturnHeader_SizeChanged(object sender, SizeChangedEventArgs e)
+        {
+            if (TargetListView != null)
+            {
+                StopAnimation();
+
+                if (IsQuickReturnEnabled)
+                {
+                    StartAnimation();
+                }
+            }
+        }
+
+        private void ScrollViewer_ViewChanged(object sender, ScrollViewerViewChangedEventArgs e)
+        {
+            if (_animationProperties != null)
+            {
+                if (!IsSticky)
+                {
+                    float oldOffsetY = 0.0f;
+                    _animationProperties.TryGetScalar("OffsetY", out oldOffsetY);
+
+                    var delta = _scrollViewer.VerticalOffset - _previousVerticalScrollOffset;
+                    _previousVerticalScrollOffset = _scrollViewer.VerticalOffset;
+
+                    var newOffsetY = oldOffsetY - (float)delta;
+
+                    // Keep values within negativ header size and 0
+                    FrameworkElement header = (FrameworkElement)TargetListView.Header;
+                    newOffsetY = Math.Max((float)-header.ActualHeight, newOffsetY);
+                    newOffsetY = Math.Min(0, newOffsetY);
+
+                    if (oldOffsetY != newOffsetY)
+                    {
+                        _animationProperties.InsertScalar("OffsetY", newOffsetY);
+                    }
+                }
+            }
+        }
+
+        private void StartAnimation()
+        {
+            // Windows.UI.Xaml.Hosting.ElementCompositionPreview is only available in Windows 10 10586 or later
+            if (!ApiInformation.IsMethodPresent("Windows.UI.Xaml.Hosting.ElementCompositionPreview", nameof(ElementCompositionPreview.GetScrollViewerManipulationPropertySet)))
+            {
+                return;
+            }
+
+            if (_scrollViewer == null)
+            {
+                return;
+            }
+
+            if (_scrollProperties == null)
+            {
+                _scrollProperties = ElementCompositionPreview.GetScrollViewerManipulationPropertySet(_scrollViewer);
+            }
+
+            var compositor = _scrollProperties.Compositor;
+
+            if (_animationProperties == null)
+            {
+                _animationProperties = compositor.CreatePropertySet();
+            }
+
+            _previousVerticalScrollOffset = _scrollViewer.VerticalOffset;
+            _headerVisual = ElementCompositionPreview.GetElementVisual((UIElement)TargetListView.Header);
+
+            _animationProperties.InsertScalar("OffsetY", 0.0f);
+
+            ExpressionAnimation expressionAnimation = compositor.CreateExpressionAnimation($"max(animationProperties.OffsetY - ScrollingProperties.Translation.Y, 0)");
```

Comment:
Yep, reproduced. With Round mouse wheel is lagging. Without Round scroll-bar is lagging.